dxr/utils.py

```diff
@@ -285,3 +285,7 @@ def is_in(needle, haystack):
 def without_ending(ending, string):
     """If ``string`` ends with ``ending``, strip it off."""
     return string[:-len(ending)] if string.endswith(ending) else string
+
+def splitlines_keepends(string):
```

Comment:
I'll surely make the recommended changes @erikrose 
and will not commit the build/ directory @pelmers